docs/GettingStarted.rst

```diff
@@ -1244,6 +1244,45 @@ You can check that the action has been done successfully by connecting to the de
 |image167|
 
 
+Pinning a News in the home page
+-------------------------------
+
+As a platform-wide publisher (publisher:/platform/users role required) If you have the membership "publisher", you are allowed to pin any articles to the home page. Pinning an article effectively publishes it from wherever it was originally posted to all users of the platform. Pin function is available at three locations described below:
+
+
+1- Pinning a News from creation form : 
+
+From simple and full creation forms, a  "Pin article" checkbox is available. 
+
+|image168|
+
+When you type in fields with the article details, tick "Pin article" checkbox then click on "post" button. A confirmation message appears.
+
+|image169|
+
+After confirmation,  the article is posted to space's activity stream and automatically published to home page News bloc. 
+
+
+2- Pinning a News from activity stream : the "pin article" option is available from the three-dots menu of the article’s activity.
+
+|image170|
+
+When you choose this option, a confirmation popup appears:
```

Comment:
```suggestion
When you select this entry, a confirmation popup appears:
```

Reply:
@samehaidi  did you deliberately decide to not accept this suggestion or did you just miss it ?

Reply:
I decided to not accept this suggestion :)

Reply:
Fine.